SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -1,32 +1,54 @@
-<%(Callout type="warn" message="This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS Unreal GDK is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
+# Port your Unreal project to the GDK
 
-# How to port a native Unreal project to the Unreal GDK
+## Before you start
 
-> This workflow is very subject to change. We're aware that it's not optimized, and one of our priorities is to improve iteration times.
+Before porting your project: 
+* Install and set up the SpatialOS GDK: follow the [set up guide]({{urlRoot}}/setup-and-installing) to the end of **Setting up the Unreal GDK module and Starter Project** > **Cloning**. 
+* Make sure your Spatial CLI is up to date: from a terminal window, run the command `spatial update`.
+<!-- // TODO: Update the set up link when ready -->
 
-## Prerequisites
+### Terms used in this guide
+`<ProjectRoot>` - The root folder of your Unreal project.  
+`<GameRoot>` - The folder containing your game project's `.uproject` and source folder (for example, `<ProjectRoot>/ShooterGame/`).  
```

Comment:
Start with defining `<GameRoot>` as the folder containing the `.uproject`.
Then, if `<GameRoot>` folder lives inside another folder, then that is the `<ProjectRoot>`, otherwise you should create a folder and move `<GameRoot>` inside of it.